cfgov/jinja2/v1/_includes/organisms/reg-comment.html

```diff
@@ -1,50 +1,95 @@
+   Creates an Regulations.gov comment form when given:
 
-   value:         An object used to customize the markup.
+   value:               An object used to customize the markup.
 
-   value.heading: A string with the title for the header slug.
+   value.document_id:   Federal Register document ID number to which
+                        the comment should be submitted.
+                        Should follow this format: `CFPB-FRDOC_####-####`.
 
-   value.text:    The text used within the description markup.
-
-   value.gd_code: A GovDelivery code for a specified mailing list.
+   value.id:            (optional) Sets the id attribute in the form's markup.
+                        If not set, the form will be assigned a base id of
+                        `o-reg-comment_` with a random number appended.
 
    ========================================================================== #}
 
-<div class="o-email-signup">
-{% if value.heading %}
-    <h2 class="header-slug">
-        <span class="header-slug_inner">
-            {{ value.heading }}
-        </span>
-    </h2>
-{% endif %}
-
-<form id="{{ value.id or ('o-email-signup_' ~ range(1, 100) | random) }}"
-      action="/subscriptions/new/"
-      method="POST"
-      enctype="application/x-www-form-urlencoded">
-    {% if value.text %}
-        <p>
-            {{ value.text }}
-        </p>
-    {% endif %}
-
-    {% set field = 'email' %}
-    {% for form in value.form_field %}
-        {{ render_stream_child(form) }}
-    {% endfor %}
-
-    <div class="form-group">
-        <input type="hidden" name="code" value="{{ value.gd_code }}">
-    </div>
-</form>
-</div>
+<div class="o-reg-comment">
+    <form id="{{ value.id or ('o-reg-comment_' ~ range(1, 100) | random) }}"
+          action="/regulation-comment/new/"
+          method="POST"
+          enctype="multipart/form-data">
+        <div class="m-form-field-with-button">
+            <div class="form-group">
+                <label for="general_comment">
+                    Enter your comments
+                </label>
+                <textarea name="general_comment" rows="5" required></textarea>
+
+                <p class="u-mb30">
+                    If you would like to include an attachment with your response,
+                    please submit your comment at
+                    <a class="icon-link icon-link__external-link"
+                       href="https://www.regulations.gov/">
+                        <span class="icon-link_text">Regulations.gov</span>
+                    </a>.
+                </p>
+            </div>
+        </div>
+
+        <div class="m-form-field-with-button">
+            <div class="form-group">
+                <label for="first_name">
+                    First name
+                </label>
+                <input type="text" id="first_name" name="first_name" required>
+            </div>
+        </div>
+
+        <div class="m-form-field-with-button">
```

Comment:
I was trying to understand the atomic structure and follow it as best as I could, but I think ultimately it doesn't end up doing anything here. Perhaps I'll remove it so future changes to that molecule don't inadvertently affect these forms.